packages/universal-test-adapter-dotnet/src/index.ts

```diff
@@ -47,14 +50,33 @@ export async function executeTests({ testsToRun = [] }: AdapterInput): Promise<A
     }
     return `(FullyQualifiedName~${parseFilepathAndClassName(filepath, suiteName)}.${testName})`
   })
+
   if (fullyQualifiedNames.length > 0) {
     args.push(`--filter`)
     args.push(`${fullyQualifiedNames.join(' | ')}`)
   }
+
+  // https://github.com/Microsoft/vstest-docs/blob/main/docs/report.md#syntax-of-default-loggers
+  switch (reportFormat) {
+    case 'default':
```

Comment:
Regarding Adam's comment from last PR, going to forgo a "junit" format for now since it needs a third-party library for dotnet -- even though our other four adapters will "default" to junit, I prefer to delay the handling of non-native report formats for the time being since I think we don't gain too much from it right now.